service-runner/src/main/resources/config/assets.yaml

```diff
@@ -5,22 +5,41 @@ assets:
     issuer: GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP
     distribution_account: GBN4NNCDGJO4XW4KQU3CBIESUJWFVBUZPOKUZHT7W7WRB7CWOA7BXVQF
     significant_decimals: 2
-    deposit:
+    sep6:
```

Comment:
Can we consider
```
  assets:
    - id: stellar:USDC::GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP
```
to replace with
```
  assets:
    - schema: stellar
      code: USDC
      issuer: GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP
```

Reply:
I don't think that's necessary. If we have the former format we will need to break it down for other SEP naming.

Beside this we are going to separate stellar asset from fiat asset (there will be 2 list like `clients` config)

Reply:
I don't think we should expect devs to use string formats defined in our SEPs. Its more human-readable to have key-val pairs for each piece of information.

Reply:
Hi @lijamie98 @JakeUrban  I want to move this discussion to the main ticket https://stellarorg.atlassian.net/browse/ANCHOR-732 to save it for future reference